leshan-lwm2m-servers-shared/pom.xml

```diff
@@ -27,7 +27,7 @@ Contributors:
   </parent>
   <artifactId>leshan-lwm2m-servers-shared</artifactId>
   <packaging>bundle</packaging>
-  <name>leshan-servers-shared</name>
+  <name>Leshan servers shared</name>
```

Comment:
```suggestion
  <name>Leshan Servers shared</name>
```

Reply:
I don't know if this is a good idea but I didn't add capital to `shared` and `servers`. Thinking that maybe this show this is not first class module.

If we decide to change add Capital to `Servers`, we probably need to also change : 
`[INFO] Leshan Demo servers shared ......................... SUCCESS [  0.099 s]` ?

Reply:
Let me know if you still want to Capitalize with this explanation ?

Reply:
After discussion we decide to only use lower case except for proper noun (Leshan, Californium, Redis)